From 64a7e402207546b53e164dc94f0ab66f9de02b5c Fri, 16 Dec 2016 14:52:44 +0100
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Wed, 5 Oct 2016 16:43:50 +0200
Subject: [PATCH] [tracker] Fix memory leak tracker-extract-libav. Fixes JB#36477


Also disable icon indexing and artwork, and own tracker/miners directory.
diff --git a/src/tracker-extract/tracker-extract-libav.c b/src/tracker-extract/tracker-extract-libav.c
index ab86e87..8c21e32 100644
--- a/src/tracker-extract/tracker-extract-libav.c
+++ b/src/tracker-extract/tracker-extract-libav.c
@@ -185,7 +185,7 @@
 	}
 
 	if (!audio_stream && !video_stream) {
-		avformat_free_context (format);
+		avformat_close_input (&format);
 		g_free (uri);
 		return FALSE;
 	}
@@ -407,7 +407,7 @@
 
 	g_free (uri);
 
-	avformat_free_context (format);
+	avformat_close_input (&format);
 
 	return TRUE;
 }
